wiki/cpp/DEVELOPER_GUIDE.md

```diff
@@ -259,6 +259,8 @@ void foo(const ML::cumlHandle_impl& h, ...)
 }
 ```
 
+An example of how to use internal streams to schedule work on a single GPU can be found in [here](https://github.com/rapidsai/cuml/pull/1015). This PR uses the internal streams inside `cumlHandle_impl` to schedule more work onto the GPU for Random Forest building.
```

Comment:
We should follow the format we've been using for the rest of the developer guide and provide the example in place. What do you think?

Reply:
Fair point. Done. Can you check now?